server/routes/activities/daily-attendance-summary/handlers/suspendedPrisoners.ts

```diff
@@ -0,0 +1,82 @@
+import { Request, Response } from 'express'
+import _ from 'lodash'
+import { convertToTitleCase, toDate } from '../../../../utils/utils'
+import ActivitiesService from '../../../../services/activitiesService'
+import AttendanceReason from '../../../../enum/attendanceReason'
+import PrisonService from '../../../../services/prisonService'
+
+export default class SuspendedPrisonersRoutes {
+  constructor(private readonly activitiesService: ActivitiesService, private readonly prisonService: PrisonService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+    const { date } = req.query
+
+    if (!date) {
+      return res.redirect('select-period')
+    }
+
+    const activityDate = toDate(req.query.date as string)
+
+    const scheduledActivities = await this.activitiesService.getScheduledActivitiesAtPrison(activityDate, user)
+    const suspendedAttendances = await this.activitiesService
+      .getAllAttendance(activityDate, user)
+      .then(r => r.filter(a => a.attendanceReasonCode === AttendanceReason.SUSPENDED))
```

Comment:
Should this be looking for AUTO_SUSPENDED status too?  SUSPENDED = manual (which we can't yet do), and AUTO_SUSPENDED means they're out of the prison and may come back.